portal-impl/test/unit/com/liferay/portal/servlet/ComboServletTest.java

```diff
@@ -273,6 +273,39 @@ public void testServiceWithPortletIdAndProxyAndContext() throws Exception {
 			_pluginServletContext);
 	}
 
+	@Test
+	public void testTooManyComboServletRequests() throws Exception {
+		MockHttpServletRequest mockHttpServletRequest =
+			new MockHttpServletRequest();
+
+		StringBuilder sb = new StringBuilder();
+
+		for (int i = 0; i < 101; i++) {
+			sb.append("/js/javascript");
+			sb.append(i);
+			sb.append(".js");
+
+			if (i != 100) {
```

Comment:
Pro Tip :tm: :sweat_smile:  : I would move this to the top of the loop and use `i != 0` in the condition. That way the code is future proof because, if you change the for loop max bound, it keeps working.

Reply:
👍

Reply:
Wait how did you think this would achieve not having `&` at the end of the string? 
I need to not add `&` if it's the last index, I don't think I care about it being the first index.

<img src="https://media3.giphy.com/media/g01ZnwAUvutuK8GIQn/giphy-downsized-medium.gif"/>

Reply:
Because you move the `if` block to the very top of the `for` block, that way you stop postfixing `&` and instead begin prefixing `&` (except for the very first item, of course).